packages/office-ui-fabric-react/tslint.json

```diff
@@ -4,6 +4,7 @@
     "deprecation": false,
     "jsx-ban-props": false,
     "no-any": false,
-    "typedef": [false]
+    "typedef": [false],
+    "import-blacklist": [true, { "../../Styling": ["FontSizes"] }]
```

Comment:
Didn't know about this rule, nice! What about ../../../Styling (for a few components) or possibly office-ui-fabric-react/lib/Styling?

Reply:
Hmmm… that’s a good question…
Let me see if I can come up with a regex

Reply:
Are we even supposed to use `office-ui-fabric-react/lib/Styling` within the package? Isn’t that somewhat an illegal import?

Reply:
It's a real module, so by the lint-imports rules at least it's legal. It's used in examples primarily.

Reply:
Yeah… but we are using it in examples because they are not part off the package bundle and we want to demo the correct import statements. I played a little with that rule but I can’t seem to be able to include a regex pattern specifically to the import location and still specify which named import to ban.
What I can do is just add a few more iterations with a couple more possible locations like the ones you mentioned. I don’t think I have enough knowledge to write my own lint rule to check for it through a regex.

Reply:
Yeah I don't think this situation justifies all the work of going through and learning how to write your own lint rule.

Reply:
Sounds reasonable. Don't block on it, we can be iterative. Just trying to help guard against future work.

Reply:
@ecraig12345 @KevinTCoughlin I have a tslint custom rule prepared that should allow us in a generic way to ban specific imports from specific modules but will prefer to have this one merged first and then open a new PR with the rule so I can get targeted feedback.

Reply:
Fine by me